docs/exploration-and-organization/delete-and-restore.md

```diff
@@ -36,7 +36,7 @@ To move an item (question, dashboard, model, or collection) to Trash:
 2. Click on the three dots menu;
 3. Select "Move to trash".
 
-> When a collection is deleted, all items in the collection are deleted as well.
+When a collection is deleted, all items in the collection are deleted as well.
```

Comment:
Do you think delete is clear here?

When I read delete I thought this was referring to "permanently delete", not "move to trash".